apps/web/src/app/(private)/settings/_components/ProviderApiKeys/index.tsx

```diff
@@ -88,6 +89,9 @@ const ProviderApiKeysTable = ({
             <TableCell>
               <Text.H4 color='foregroundMuted'>{apiKey.token}</Text.H4>
             </TableCell>
+            <TableCell>
```

Comment:
Don't show it. Is more confusing for people that just use OpenAi or other hosted providers. They don't know about this